com.unity.formats.fbx/Editor/ExportModelSettings.cs

```diff
@@ -1,25 +1,36 @@
 ﻿using UnityEditor;
 using UnityEngine;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.IO;
```

Comment:
Why did you need to add these (I'm probably just not seeing it)

Reply:
I was initially using a different method for copying the export settings, but forgot to remove these using statements.